src/som/compiler/MixinBuilder.java

```diff
@@ -607,8 +611,10 @@ public static SSymbol getInitializerName(final SSymbol selector) {
   }
 
   public static SSymbol getInitializerName(final SSymbol selector,
-      final int mixinId) {
-    return symbolFor("initializer`" + mixinId + "`" + selector.getString());
+      final int mixinId, final MixinDefinitionId mixinDefinitionId) {
+    return symbolFor(
+        "initializer`" + mixinId + "`" + "_" + mixinDefinitionId + "_"
```

Comment:
Do we really need both, `mixinId` and `mixinDefinitionId`?

Reply:
mixinId is needed to distinguish multiple mixins used by the same class

Reply:
Ah, and please stick with a single style of separators.
For some reason, I decided that ` is good, probably, because it can't be part of a normal identifier while `_` can.